contrib/IECoreUSD/src/IECoreUSD/SceneCacheData.cpp

```diff
@@ -675,20 +675,21 @@ void SceneCacheData::loadPrimVars( const SceneInterface::Path& currentPath, TfTo
 
 	variablesPath.insert( variablesPath.end(), g_staticIoVariablesPath.begin(), g_staticIoVariablesPath.end() );
 
-	if ( auto variables = m_sceneio->directory( variablesPath, IndexedIO::MissingBehaviour::NullIfMissing ) )
+	if ( const auto& variables = m_sceneio->directory( variablesPath, IndexedIO::MissingBehaviour::NullIfMissing ) )
```

Comment:
I forgot that I added a reference here. It probably isn't too expensive to copy a `ConstIndexedIOPtr` and I didn't update other call sites. Maybe I should remove it to remain consistent with the rest of the changes?

Reply:
Ah yes, good point.
It hasn't been an issue so far but it would be good to fix it at some point.
Up to you if you want to do it now or later.

Reply:
I've had a longer think about this situation and I don't believe there is any performance benefit to using the const reference in this case. Since we are declaring `variables` and immediately assigning to it in the same statement, the compiler should elide the copy and we should end up with the same performance since there shouldn't be any incrementing or decrementing of the reference count. So, I'm going to remove the reference so that it fits the rest of the code's style.